File: API/PCPartPicker/Models/Cpucooler.cs
using System;
using System.Collections.Generic;

namespace PCPartPicker.Models;

public partial class Cpucooler
{
    public int Id { get; set; }

    public string? Name { get; set; }

    public decimal? Price { get; set; }

    public decimal? Rpm { get; set; }

    public decimal? NoiseLevel { get; set; }

    public string? Color { get; set; }

    public decimal? Size { get; set; }

    public int? ManufacturerTypeId { get; set; }

    public virtual ICollection<Configuration> Configurations { get; set; } = new List<Configuration>();

    public virtual ManufacturerType? ManufacturerType { get; set; }
}

File: UI/pc-part-picker/src/model/Cpucooler.ts
import { ManufacturerType } from "./ManufacturerType";
import { PcPart } from "./Pcpart";

export interface Cpucooler extends PcPart{
    rpm: number | null;
    noiseLevel: number | null;
    color: string | null;
    size: number | null;
    manufacturerTypeId: number | null;
    manufacturerType: string | null;
}